backend/api/simulation.py

```
@@ -1766,6 +1794,153 @@ def is_main_character_submit(persona_name, student_role):
+                        scene_id=next_scene_id,
+                        scene_completed=True,
+                        next_scene_id=next_scene_id,
+                        persona_name="System",
+                        persona_id=None,
+                        turn_count=0,
+                        scene_intro_message=scene_intro_message
+                    )
+                else:
+                    # No more scenes - simulation complete
+                    user_progress.simulation_status = "completed"
+                    user_progress.completed_at = datetime.utcnow()
```

Ruff (0.14.1)

`datetime.datetime.utcnow()` used

(DTZ003)